packages/fe-mockserver-core/src/router/serviceRouter.ts

```diff
@@ -64,8 +64,17 @@ export async function serviceRouter(service: ServiceConfigEx, dataAccess: DataAc
         req.tenantId = tenantId;
 
         parser(req, res, function () {
-            (req as any).body = (req as any).body.toString('utf-8');
-            if (req.headers['content-type'] === 'application/json') {
+            if (typeof (req as any).body === 'object' && Object.keys((req as any).body).length === 0) {
```

Comment:
didn't check: Can `req.body` be null? If yes, this will fail

Reply:
not sure also but i'm putting a check for it

Reply:
done